Dockerfile

```diff
@@ -181,15 +184,24 @@ RUN echo "Installing sdk tools ${ANDROID_SDK_TOOLS_VERSION}" && \
     rm -f flutter.tar.xz && \
     flutter config --no-analytics
 
-
 # Copy sdk license agreement files.
 RUN mkdir -p $ANDROID_HOME/licenses
 COPY sdk/licenses/* $ANDROID_HOME/licenses/
 
 # Create some jenkins required directory to allow this image run with Jenkins
-RUN mkdir -p /var/lib/jenkins/workspace && \
-    mkdir -p /home/jenkins && \
-    chmod 777 /home/jenkins && \
-    chmod 777 /var/lib/jenkins/workspace && \
-    chmod 777 $ANDROID_HOME/.android
-
+RUN mkdir -p /var/lib/jenkins/workspace
+RUN mkdir -p /home/jenkins
+RUN chmod 777 /home/jenkins
+RUN chmod 777 /var/lib/jenkins/workspace
+RUN chmod 777 $ANDROID_HOME/.android
```

Comment:
Please use single `RUN` to reduce docker layers

Reply:
@mingchen Done.